src/client/MapleClient.java

```diff
@@ -121,6 +121,7 @@
 	private final Lock lock = MonitoredReentrantLockFactory.createLock(MonitoredLockType.CLIENT, true);
         private final Lock encoderLock = MonitoredReentrantLockFactory.createLock(MonitoredLockType.CLIENT_ENCODER, true);
         private static final Lock loginLocks[] = new Lock[200];  // thanks Masterrulax & try2hack for pointing out a bottleneck issue here
+    private Calendar tempBanCalendar;
```

Comment:
Is this indented properly?

Reply:
Looks like the rest of the field variables in `MapleClient` use tabs instead of spaces, unlike the majority of the source.